lutris/gui/application.py

```diff
@@ -395,6 +433,42 @@ def do_command_line(self, command_line):  # noqa: C901  # pylint: disable=argume
             self.generate_script(db_game, options.lookup_value("output-script").get_string())
             return 0
 
+        # check if path is provided with install
+        if (options.contains("bin_path") or options.contains("install_path")
+           or options.contains("install_input_menu")) and not options.contains("install"):
+            self._print(command_line, _("No installer provided!"))
+            return 1
+
+        if action == "unattended_install":
+            self.activate()
+            install_path = None
+            input_menu = []
+            if options.contains("install_path"):
+                install_path = options.lookup_value("install_path").get_string()
+            if options.contains("input_menu"):
+                input_menu = options.lookup_value("input_menu").get_string().split(":")
+
+            unattended_install.UnattendedInstall(
+                game_slug=game_slug,
+                installer_file=installer_file,
+                revision=revision,
+                binary_path=options.lookup_value("bin_path").get_string().split(":"),
```

Comment:
you're check (above) for `install_path` and `input_menu` but for `bin_path` no if statement ... is it safe ?